src/components/impact/styledImpact.js

```diff
@@ -0,0 +1,108 @@
+import styled from "styled-components";
+
+export const ImpactWrapper = styled.section`
+    background: #F4F4F4;
+    padding: 120px 0;
+    display: flex;
+    flex-direction: column;
+`
+export const BoxWrapper = styled.div`
+    display: flex;
+    justify-content: space-around;
```

Comment:
I can't tell how big the space between the 2 boxes is but it looks larger than 40px to me.

I think you should consider putting a width on this element then changing justify-content to 'space between'. I think this makes more sense because 'margin: auto (for the left and right)' is doing the same thing as 'justify-content: space-around'.